src/main/kotlin/com/atlassian/performance/tools/report/api/junit/JUnitReport.kt

```diff
@@ -2,12 +2,14 @@ package com.atlassian.performance.tools.report.api.junit
 
 import com.atlassian.performance.tools.io.api.ensureDirectory
 import com.atlassian.performance.tools.io.api.resolveSafely
+import com.atlassian.performance.tools.jiraactions.api.ActionType
 import java.nio.file.Path
 
 interface JUnitReport {
 
     val testName: String
     val successful: Boolean
+    val action: ActionType<*>? get() = null
```

Comment:
This breaks source compatibility for Java implementors.

We can introduce a new subinterface: `ActionJunitReport: JUnitReport` and add the member there. Action-aware judges can produce such "enriched" reports. 

PS. judges offer a `Verdict`, which does not offer the reports at this point.

Reply:
Cool idea with the subinterface.
Yeah, will have to add functionality to `Verdict` as well.